services/identity/struct.go

```diff
@@ -221,7 +221,7 @@ type ProposeVote struct {
 
 // PropagateIdentity sends a new identity to other identityServices
 type PropagateIdentity struct {
-	*storage
+	*Storage
```

Comment:
Question: Why is this exported? I didn't see a usage outside of the `identity`package.